src/multibody/fcl.hpp

```diff
@@ -110,15 +110,19 @@ struct GeometryObject
   /// \brief Absolute path to the mesh file
   std::string meshPath;
 
+  /// \brief Scaling vector applied to the fcl object.
+  Eigen::Vector3d meshScale;
+
   GeometryObject(const std::string & name, const FrameIndex parentF,
                  const JointIndex parentJ, const boost::shared_ptr<fcl::CollisionGeometry> & collision,
-                 const SE3 & placement, const std::string & meshPath)
+                 const SE3 & placement, const std::string & meshPath, const Eigen::Vector3d & meshScale)
```

Comment:
There should be a initial value for `scale` and `meshPath` I think. It is very annoying in test files to have to add `"", vector` at the end...